skyportal/handlers/api/comment.py

```diff
@@ -1072,12 +1082,23 @@ def get(self, associated_resource_type, resource_id, comment_id):
             )
 
             if download:
+                attachment = decoded_attachment
+
+                if preview and (
+                    attachment_name.endswith("fit") or attachment_name.endswith("fits")
```

Comment:
```suggestion
                    attachment_name.lower().endswith(("fit", "fits"))
```

Reply:
You may also be able to get rid of the parenthesis after the `and` because there isn't an `or` anymore.

Reply:
Done.